asset/css/schedule-element.less

```diff
@@ -0,0 +1,164 @@
+
+      &:first-of-type {
+        margin-right: 1em;
+      }
+
+      &:disabled {
+        color: @schedule-element-fields-disabled-color;
+      }
+    }
+  }
+
+  .radio-label {
+    width: 100%;
+    margin-bottom: .5em;
+    display: flex;
+    align-items: center; // To center the radio element on safari
+  }
+
+  .number-specifier > input[type="number"] {
+    width: 5em;
+    margin: 0 1em;
+  }
+
+  .monthly, .ordinal:not(.ordinal-annually) {
+    padding: .5em;
+    margin-left: -.5em;
+    border: 1px solid @schedule-element-fields-border-color;
+    .rounded-corners(.75em);
+  }
+
+  .schedule-element-fields {
+    margin: 0;
+    padding: 0;
+    .rounded-corners(.25em);
+    overflow: hidden;
+
+    &.disabled { // When the "On the" radio button is checked
+      pointer-events: none;
+
+      label {
+        color: inherit;
+        background-color: @schedule-element-fields-disabled-bg;
+      }
+
+      input:checked + label {
+        background: @schedule-element-fields-disabled-selected-bg;
+        color: @schedule-element-fields-disabled-color;
+      }
+    }
+
+    li {
+      list-style: none;
+      display: inline-flex;
+      width: calc(100% / 7);
+
+      label {
+        width: 100%;
+        cursor: pointer;
+        text-align: center;
+        padding: .8em 0 .8em;
+        background: @schedule-element-fields-bg;
+        color: @schedule-element-text-color;
+
+        &:hover {
+          background-color: @schedule-element-fields-hover-bg;
+        }
+      }
+
+      &:not(:last-child) label {
+        border-right: 1px solid @schedule-element-fields-border-color;
+      }
+
+      input:checked + label {
+        background-color: @schedule-element-fields-selected-bg;
+        color: @schedule-element-default-text-color;
+      }
+
+      input:checked + label:hover {
+        background-color: @schedule-element-fields-selected-hover-bg;
+        border-color: @schedule-element-fields-selected-hover-bg;
+      }
+    }
+
+    &.multiple-fields {
+      input:focus + label {
+        box-shadow: inset 0 0 0 3px rgba(0, 195, 237, 0.5);
+      }
+
+      input:checked:focus + label {
+        box-shadow: inset 0 0 0 3px rgba(255, 255, 255, .5);
+      }
+    }
+  }
+
+  .monthly {
+    li label {
+      border-top: 1px solid @schedule-element-fields-border-color;
+    }
+
+    li:first-child,
+    li:nth-child(2),
+    li:nth-child(3),
+    li:nth-child(4),
+    li:nth-child(5),
+    li:nth-child(6),
+    li:nth-child(7) {
+      label {
+        border-top: none;
+      }
+    }
+
+    li:nth-child(7n) label {
+      border-right: none;
+    }
+  }
+
+  .annually {
+    li {
+      width: 25%; // 100% / 4 elements
+    }
+
+    li label {
+      .rounded-corners(.25em);
+      margin-right: 1px;
+      margin-bottom: 1px;
+    }
+
+    .schedule-element-fields li label {
+      border-right: none;
+    }
+
+    .schedule-element-fields:focus-within {
+      outline: 3px solid rgba(0, 195, 237, 0.5);
```

Comment:
Can we outsource the rgba value to a variable?

Reply:
I used a function to fade the `@base-primary-bg`.